tensorflow/core/kernels/topk_op_gpu.h

```diff
@@ -336,12 +330,19 @@ __device__ void mergeShards(int num_shards, int k,
   }
 }
 
-extern __shared__ char shared_memory[];
+#if GOOGLE_CUDA
+__shared__ char shared_memory[];
```

Comment:
Upstream has the keyword `extern`. The way it is now will not pass cuda compilation. Mind pulling the changes from upstream in a new PR?

Reply:
My bad.
It can't be upstreamed as is because it depends on at least one other PR that has not been upstreamed yet. It is better to open a new PR here.

Reply:
Gotcha, no worries. I was running with something else on Cuda and happened to notice this. Work on it at your convenience. Thanks